src/interface/src/app/backend-constants.ts

```diff
@@ -0,0 +1,3 @@
+export class BackendConstants {
+  public static Host: string = "http://localhost:8000"
```

Comment:
This should be all uppercase to denote it's a constant

Reply:
done.